modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -95,6 +95,8 @@ Ensure that you have the latest available version of this tool to use all the fu
 Download link: https://docs.mulesoft.com/downloads/mule-runtime/4.2/secure-properties-tool.jar[Secure Properties Tool Jar file]. +
 Latest release: 09/09/2020.
 
+Note that the syntax has changed from earlier versions, to avoid issues use the syntax and the JAR file provided in this article.
```

Comment:
Minor suggestion: "Note that the syntax has changed from earlier versions**. T**o avoid issues, use the syntax and the JAR file provided in this article."